content/concepts/payment-system-basics/accounts/accounts.md

```diff
@@ -75,6 +75,8 @@ To be deleted, an account must meet the following requirements:
 
 After an account has been deleted, it can be re-created in the ledger through the normal method of [creating accounts](#creating-accounts). An account that has been deleted and re-created is no different than an account that has been created for the first time.
 
+**Warning:** The [AccountDelete transaction][]'s transaction cost always applies when the transaction is included in a validated ledger, even if the transaction failed because the account does not meet the requirements to be deleted. To greatly reduce the chances of paying the high transaction cost if the account cannot be deleted, [submit the transaction](submit.html) with `fail_hard` enabled.
```

Comment:
Would it be worth mentioning the `"deletion_blockers_only" : "true"` to the `account_objects` method here?